_rules/iframe-identical-name-equivalent-purpose-4b1c6c.md

```diff
@@ -130,6 +136,59 @@ Two `iframe` elements within the same [document tree][] have the same [accessibl
 <iframe title="advertising" src="/test-assets/iframe-unique-name-4b1c6c/advertising-two.html"> </iframe>
 ```
 
+#### Passed Example 9
+
+All three `iframe` elements have the same [accessible name][]. The second `iframe` (with `id` `"light"` ) is only part of the [light tree][]. When the [shadow tree][] is attached to `host` and flattened, this `iframe` is overwritten and therefore not part of the [flat tree][]. Hence, only the first and third `iframe` are considered by this rule and they both point to the [same resource][].
```

Comment:
I think you've gone a little over board on the shadow DOM testing. These tests are intended to check if this iframe rule was implemented correctly, not if shadow DOM was correctly implemented.

Reply:
Yes, possibly 🙄 
I've been putting here essentially the same examples as in #768. Which might still be too much… 

=> can we move the discussion concerning which examples to keep to #768 (which is in Final Call). I'll reflect the changes here.

Reply:
That's fine, but I'm still leaving this on changes requested. This is testing the wrong thing.